modos/api.py

```diff
@@ -196,6 +197,16 @@ def remove_element(self, element_id: str):
 
         zarr.consolidate_metadata(self.zarr.store)
 
+    def remove_object(self):
+        """Remove the complete modo object"""
+        for fi in self.list_files():
+            self.storage.remove(fi)
+        self.zarr.store.rmdir()
```

Comment:
**praise:** very clean and satisfying, good that we abstracted this away :)

Reply:
**comment:** also clean to do the deletion with list_files() directly. No risk of deleting any external files (when that will be possible).